docs/migration/00b_Electronic_structure_with_v0.5.ipynb

```diff
@@ -0,0 +1,729 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "5716043f",
+   "metadata": {},
+   "source": [
+    "# Electronic Structure Problems with v0.5\n",
+    "\n",
+    "## Table of Contents\n",
+    "The electronic structure stack has received the most updates between Qiskit Nature v0.4 and v0.5. Thus, this file is split into multiple sections that go into various amounts of details. Here is a table of contents to help you navigate:\n",
+    "\n",
+    "- [TL;DR](#TL;DR)\n",
+    "- [qiskit_nature.drivers](#qiskit_nature.drivers)\n",
+    "- [qiskit_nature.transformers](#qiskit_nature.transformers)\n",
+    "- [The ElectronicStructureProblem (qiskit_nature.problems)](#The-ElectronicStructureProblem-(qiskit_nature.problems))\n",
+    "- [qiskit_nature.properties](#qiskit_nature.properties)\n",
+    "\n",
+    "## Further resources\n",
+    "Be sure to check out the following tutorials for how to use the new code:\n",
+    "\n",
+    "- [Electronic Structure Problems](../tutorials/01_electronic_structure.ipynb)\n",
+    "- [Problem Transformers](../tutorials/05_problem_transformers.ipynb)\n",
+    "- [The QCSchema](../tutorials/08_qcschema.ipynb)\n",
+    "- [Properties](../tutorials/09_properties.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "21b0aeb0",
+   "metadata": {},
+   "source": [
+    "## TL;DR\n",
+    "\n",
+    "This section gives you one cell with a v0.4 code followed by one cell with v0.5 code doing the same things.\n",
+    "Hopefully this already gives you all the information which you were looking for."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "467062e5",
+   "metadata": {},
+   "source": [
+    "### Previously"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "5440409a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "<ipython-input-1-aea959c80ea0>:7: NatureDeprecationWarning: The qiskit_nature.problems.second_quantization.electronic package is deprecated as of version 0.5.0 and will be removed no sooner than 3 months after the release. Instead use the qiskit_nature.second_q.problems package.\n",
+      "  from qiskit_nature.problems.second_quantization import ElectronicStructureProblem\n",
+      "<ipython-input-1-aea959c80ea0>:7: NatureDeprecationWarning: The qiskit_nature.problems.second_quantization.vibrational package is deprecated as of version 0.5.0 and will be removed no sooner than 3 months after the release. Instead use the qiskit_nature.second_q.problems package.\n",
+      "  from qiskit_nature.problems.second_quantization import ElectronicStructureProblem\n"
```

Comment:
It seems strange in the notebook about ElectronicStructure for this to have an message about vibrational. Maybe its the way we have things imported in the init files but I would this it would seem odd to a user.

Also is this all I would expect to see - e.g. its using an older PySCF driver and seemingly no message about that or the Molecule etc

Reply:
That might go into how the deprecation warnings are being handled.

@manoelmarques
- do you have any insights why we see the vibrational one appearing here where no vibrational parts are being used?
- and what about the deprecation of other parts like driver, transformer, etc.?